github-sectory

```diff
@@ -94,14 +94,20 @@ class GithubSectory():
 		puts(colored.yellow("[ Getting folder content ]"))
 		self.downloadLink = parameter
 		try:
-			self.parsed = json.loads(urllib2.urlopen(self.downloadLink).read())
+			self.parsed = json.loads(urllib2.urlopen(self.downloadLink, timeout=30).read())
```

Comment:
What if the file size is not too high, but due to the slow connection it exceeds the time limit then? 

I think we should not include the `timeout`. Let me know your thoughts on this?